### DISCUSSION TITLE
OpenRV supports ARM based Macs?

### DISCUSSION THREAD

**Discussion by AlsonEntuna** [AcademySoftwareFoundation/OpenRV#377] [Q&A] [answered]
Hi everyone,

Has anyone of you tried compiling openrv using arm macs? The M-series mac? because currently I only have a M1 Mac and I'm curious if we support this architecture when compiling for mac?

**Comment by geffrak** [comment, marked as answer]
The current build is an "intel" build, no matter what you have as CPU. If you are running with Apple Silicon, Rosetta is required.

There's a task in the roadmap to have a native Apple Silicon build, and we already did some explorations about what's required to have it working.

**Reply by AlsonEntuna** [reply]
so I can still technically can compile it on a m1 mac and run it right? thanks for the answer!

**Reply by geffrak** [reply]
Yes! My main computer is a M1 Pro MacBook and it works pretty well.

**Reply by AlsonEntuna** [reply]
alrigh! thank you so much for answering
